index.html

```diff
@@ -44,6 +44,158 @@ <h2 class="name1">I’am a Software Developer</h2>
+         <br>
+       <div class="tags1">
+         <ul>
+           <li class="tag html1">html</li>
+           <li class="tag bootstrap1">bootstrap</li>
+           <li class="tag ruby1">Ruby</li>
+         </ul>
+       </div>
+         <br>
+         <button class="projects1">See project</button>
+     </div>
+     </div>
+
+     <div class="showcaseQ">
+       <div class="Art">
+        <br><br><br>
+         <h2 class="stories1"> Profesional Art Printing Data</h2>
+         <p class="paragraph1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+         <br>
+         <div class="tags1">
+           <ul>
+             <li class="tag html1">html</li>
+             <li class="tag bootstrap1">bootstrap</li>
+             <li class="tag ruby1">Ruby</li>
+           </ul>
+         </div>
+           <br>
+           <button class="projects1">See project</button>
+       </div>
+       </div>
+   </section>
+   <section class="about">
+       <h2 class="myself">About me</h2>
+       <p id="paragraph">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+       <button class="get">Get My Resume</button>
+       <img class="pics" src="img/illustration-about-me-mobile.png" alt="">
+       <li class="barQk"><div id="barQk"></div></li>
+
+    <article>
+        <div class="LANGTTT">
+          <ul id="list">
+            <li class="list"><img src="img/Rectangle 55@3x.png" alt="" class="image" >
+             <p class="langQ">Language</p></li>
+            <li class="lang1">Javascript</li>
+            <li class="lang2">Ruby</li>
+            <li class="lang3">Html</li>
+            <li class="lang4">Css</li>
+          </ul>  
+      </div>
+    </article>
+
+      <article>
+        <div class="LANGTTT">
+           <ul id="list">
+             <li class="listFRAM"><img src="img/Rectangle 55@3x.png" alt="" class="image4" >
+               <p class="langQ1">Frameworks</p></li>
+             <li class="lang1">Bootstrap</li>
+             <li class="lang2">Ruby on Rails</li>
+             <li class="lang3">RSPec</li>
+             <li class="lang4">CapyBara</li>
+             <li class="lang4">Selenium</li>
+           </ul>  
+        </div>
+    </article>
+
+   <article>
+      <div class="LANGTTT1">
+         <ul id="list">
+          <li class="listSKL"><img src="img/Rectangle 57.png" alt="" class="image5" >
+            <p class="langQ">Skills</p></li>
+           <li class="lang1">Codekit</li>
+           <li class="lang2">Github</li>
+           <li class="lang3">CodePen</li>
+           <li class="lang4">Gitlab</li>
+           <li class="lang4">Terminal</li>
+         </ul>  
+       </div>
+    </article>
+   </section>
```

Comment:
- [ ] Your design 👇 
![image](https://user-images.githubusercontent.com/78251927/205436167-7b393ef7-2acc-4d13-9dcb-a87044d5688c.png)

- [ ] Figma design 👇 
![image](https://user-images.githubusercontent.com/78251927/205436418-517bd1bc-9711-4b1f-98a3-a35cf0fb3a8c.png)

- [ ] As requested by the previous reviewers, and according to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), it is important to stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

- [ ] Please remove the (margin/padding) around the **"About me"** section to match the Figma design 👏